source/_components/sensor.growatt.markdown

```diff
@@ -0,0 +1,37 @@
+---
+layout: page
+title: "Growatt sensor"
+description: "Instructions on how to integrate a Growatt solar panel plant into Home Assistant."
+date: 2019-01-30 15:04
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: growatt.png
+ha_category: Energy
+ha_release: 0.88.0
```

Comment:
Maybe add : 

```
:ha_iot_class: "Cloud Polling"
```

Reply:
👍 , what does it do though?

Reply:
https://www.home-assistant.io/blog/2016/02/12/classifying-the-internet-of-things/